lib/lltz_michelson/lltz_michelson.ml

```diff
@@ -280,4 +281,61 @@ and compile_assign (var : string) value =
       Instruction.trace (compile value);
       Instruction.Slot.set (`Ident var)
     ]
-  )
\ No newline at end of file
+  )
+
+(* Compile an if-bool expression by compiling the condition, then applying the if-bool instruction to the condition and the true and false branches. *)
+and compile_if_bool condition if_true if_false =
+  Instruction.seq [
```

Comment:
let open Instruction in ...
(Same for the other functions.) I think such opening is acceptable here.

Reply:
is it ok to just open Instruction in the beginning of the file as it will be used by almost every function?